pkg/sdk/databases.go

```diff
@@ -139,17 +138,22 @@ func (row databaseRow) convert() *Database {
 
 // CreateDatabaseOptions is based on https://docs.snowflake.com/en/sql-reference/sql/create-database.
 type CreateDatabaseOptions struct {
-	create                     bool                    `ddl:"static" sql:"CREATE"`
-	OrReplace                  *bool                   `ddl:"keyword" sql:"OR REPLACE"`
-	Transient                  *bool                   `ddl:"keyword" sql:"TRANSIENT"`
-	database                   bool                    `ddl:"static" sql:"DATABASE"`
-	IfNotExists                *bool                   `ddl:"keyword" sql:"IF NOT EXISTS"`
-	name                       AccountObjectIdentifier `ddl:"identifier"`
-	Clone                      *Clone                  `ddl:"-"`
-	DataRetentionTimeInDays    *int                    `ddl:"parameter" sql:"DATA_RETENTION_TIME_IN_DAYS"`
-	MaxDataExtensionTimeInDays *int                    `ddl:"parameter" sql:"MAX_DATA_EXTENSION_TIME_IN_DAYS"`
-	Comment                    *string                 `ddl:"parameter,single_quotes" sql:"COMMENT"`
-	Tag                        []TagAssociation        `ddl:"keyword,parentheses" sql:"TAG"`
+	create                     bool                     `ddl:"static" sql:"CREATE"`
+	OrReplace                  *bool                    `ddl:"keyword" sql:"OR REPLACE"`
+	Transient                  *bool                    `ddl:"keyword" sql:"TRANSIENT"`
+	database                   bool                     `ddl:"static" sql:"DATABASE"`
+	IfNotExists                *bool                    `ddl:"keyword" sql:"IF NOT EXISTS"`
+	name                       AccountObjectIdentifier  `ddl:"identifier"`
+	Clone                      *Clone                   `ddl:"-"`
+	DataRetentionTimeInDays    *int                     `ddl:"parameter" sql:"DATA_RETENTION_TIME_IN_DAYS"`
+	MaxDataExtensionTimeInDays *int                     `ddl:"parameter" sql:"MAX_DATA_EXTENSION_TIME_IN_DAYS"`
+	ExternalVolume             *AccountObjectIdentifier `ddl:"identifier,equals" sql:"EXTERNAL_VOLUME"`
+	Catalog                    *AccountObjectIdentifier `ddl:"identifier,equals" sql:"CATALOG"`
+	DefaultDDLCollation        *string                  `ddl:"parameter,single_quotes" sql:"DEFAULT_DDL_COLLATION"`
+	LogLevel                   *LogLevel                `ddl:"parameter,single_quotes" sql:"LOG_LEVEL"`
+	TraceLevel                 *TraceLevel              `ddl:"parameter,single_quotes" sql:"TRACE_LEVEL"`
+	Comment                    *string                  `ddl:"parameter,single_quotes" sql:"COMMENT"`
+	Tag                        []TagAssociation         `ddl:"keyword,parentheses" sql:"TAG"`
```

Comment:
Is `TAG` and `WITH TAG` the same?

Reply:
Yes, as you can see in the syntax section for any object, it's presented as `[ WITH ] TAG` meaning the WITH is optional.